### PR TITLE
Setting up VM for Hardware Emulation

### PR DESCRIPTION
**Pull request by Shelby-dotcom**
Documentation for setting up hardware emulation on VM

### CODE REVIEW

File: doc/GettingStarted.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Getting started is easy. We'll show you a few commands to get your feet wet, tes
 
   - jump right into development with help from the [Developer's Guide](DevelopersGuide.md#top), or
   - test drive AWS F1 with [Getting Started with F1](GettingStartedF1.md#top)
+  - Alternatively, if you want to setup development environment for Hardware Emulation on your compatible machine, read [Setting up Virtual Machine for Hardware Emulation](VMsetup.md#setting-up-virtual-machine-for-hardware-emulation)
```
Comment (stevehoover) [inline doc/GettingStarted.md:10]: "setup" -> "set up the"
Remove the # anchor and just reference the doc.

File: doc/VMsetup.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<a name="VM"></a>
+# Setting up Virtual Machine for Hardware Emulation
```
Comment (stevehoover) [inline doc/VMsetup.md:2]: This is specifically for Windows, right?

File: doc/GettingStarted.md (hunk 1 above)
Comment (stevehoover) [reply doc/GettingStarted.md:10]: Windows, right?

File: doc/VMsetup.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+## Overview
+The focus of this document is to list down a number of steps which a user can opt for to setup a virtual machine specifically to do Hardware emulation. At the end of these steps, your VM is exactly like c4.4x EC2 instance and you can emulate your kernel on hardware just like AWS's Alveo Accelerator Card.
```
Comment (stevehoover) [inline doc/VMsetup.md:5]: Summarize the steps. Something like:
  - Install Virtual Box
  - Install Vitis
  - Install XRT
If any of these steps are adequately documented elsewhere, just link to instructions to reduce maintenance.

File: doc/VMsetup.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
```
Comment (stevehoover) [inline doc/VMsetup.md:6]: State the assumptions of these instructions, e.g. date they were last tested, F1 machine config at this time: OS, Xilinx version.

**Review by stevehoover — APPROVED** [review]

**Comment by stevehoover** [comment]
Though merged, please address comments. Thx!

File: doc/VMsetup.md (hunk 2 above)
Comment (Shelby-dotcom) [reply doc/VMsetup.md:2]: Yes, its for windows. Let me add it in doc

File: doc/VMsetup.md (hunk 3 above)
Comment (Shelby-dotcom) [reply doc/VMsetup.md:5]: sure, i can add this list in overview section and point it to respective sections

**Pull request merged**